fls-shake.nix

```
@@ -0,0 +1,27 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+let
+  ghcEnv = pkgs.haskellPackages.ghcWithPackages (hp: [
+    hp.shake
+    hp.hashable
+    hp.binary
+    hp.deepseq
+  ]);
+in
+pkgs.stdenv.mkDerivation {
+  pname = "fls-shake";
+  version = "0.1.0.0";
+
+  src = ./.;
```

Use `fileset` to only depend on `Shakefile.hs`:
```suggestion
  src = pkgs.fileset.toSource {
      root = ./.;
      fileset = ./Shakefile.hs;
    };
```